### ISSUE TITLE
[NEW FEATURE]  Command to demand the nodes' Bootloader Mode 

### ISSUE THREAD

**Issue by BiliouriV** [Luos-io/luos_engine#404]
## Before to begin

Thank you for contributing to the Luos project!

Before to begin, please ensure that the feature you are requesting doesn't exist yet (you can use the GitHub searchbar with keywords to look for other issues).

If you only have a usage question, ask it to the [Luos Community](https://discord.gg/luos).

At last, feel free to read the [Luos contribution's guidelines](https://github.com/Luos-io/luos_engine/blob/main/CONTRIBUTING.md) and the [documentation page](https://www.luos.io/docs/contribute-to-luos) to have more insight about how to contribute to Luos!

## Details
Please replace the italic test by your inputs and details in each of the following sections:

### What feature would you like to be included in Luos?
It would be nice to have a message sent from Pyluos  and Gate in order to demand each node's bootloader mode.

### Describe the expected behavior of the feature
Pyluos before starting the uploading procedure should verify the bootloader mode of each node. We should be able to check that the mode is in APP_RELOAD_MODE in order to start the procedure.

### What is the motivation / use case for changing the behavior?
In case that we have a busy network with a lot of messages like updates, it is possible that the first message when we want to reflash the boards is not received by the corresponding board (Collisions etc). For this reason we need to verify if the message is received and resend it if not.

Thank you! Your issue will be reviewed by our admins.
